src/db/drizzle.provider.ts

```diff
@@ -25,8 +25,5 @@ export class DrizzleProvider implements OnModuleInit {
     });
     this.db = drizzle(pool, { schema });
     migrate(this.db, {migrationsFolder: './src/db/migrations'});
```

Comment:
```suggestion
    await migrate(this.db, { migrationsFolder });
```

and at the top of the file add something like

```ts
import path from 'path';

const migrationsFolder = path.join(__dirname, 'migrations');
```

Reply:
this is because we can't assume to have the migrations on SRC when doing the production build

Reply:
@ulises-jeremias  #bayerLessons

Reply:
Fixed!